doc/users/dflt_style_changes.rst

```diff
@@ -647,7 +647,7 @@ The color of the lines in the hatch is now determined by
   is looked up at artist creation time.
 
 The width of the lines in a hatch pattern is now configurable by the
-rcParams `hatch.linewidth`, which defaults to 1 point.  The old
+rcParams :rc:`hatch.linewidth`, which defaults to 1 point.  The old
```

Comment:
```suggestion
rcParams :rc:`hatch.linewidth`.  The old
```
Default is automatically given.

Reply:
would rather leave the unrendered docs readable and meaningful even if we later change the default (here the default really matters).

Reply:
Reads a bit strange now:

> ... is now configurable by the rcParams rcParams["hatch.linewidth"] (default: 1.0), which defaults to 1 point. 

https://11115-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/users/dflt_style_changes.html#hatching

Maybe don't use `:rc:` here?

Reply:
sure